packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -297,6 +299,122 @@ curl -XPUT 'https://${yourOktaDomain}/api/v1/myaccount/directoryProfile' -H 'Aut
+| `retry`  | Boolean | An optional property that indicates whether this is a normal challenge or retry |
+
+#### Response body
+
+Returns an empty response with an HTTP 200 status code.
+
+Passing an invalid `method` returns a 400 BAD REQUEST with error code E0000001.
+Disabling the factor type of the corresponding method on the org returns a 403 FORBIDDEN with error code E0000038.
+Passing an invalid `id` returns a 404 NOT FOUND with error code E0000008.
+Call providers failing to send the out of band OTP challenge returns 500 with error code E0000138.
+
+#### Usage example
+
+##### Request
+
+This request sends a verification code by SMS to the phone number represented by `id`. The request is a normal phone challenge, not a retry.
+
+```bash
+curl -XPOST 'https://${yourOktaDomain}/myaccount/phones/{id}/challenge' -H 'Authorization: bearer {token}' -H 'Content-Type: application/json' --data '{
+     "method": "SMS",
+     "retry": false
+ }'
+```
+
+### Verify Phone Challenge
+
+<ApiOperation method="post" url="/idp/myaccount/phones/{id}/verify"/>
+
+Verify the phone number with the verification code that the user receives via `SMS` or `CALL`. The phone number is active upon a successful verification.
+
+> **Note:** Sending requests to the `/idp/myaccount/phones/{id}/verify` endpoint at a rate that exceeds the rate limit rule configured by the admin, returns a 429 (Too Many Requests) error.
+
+#### Required scope and role
+
+An Okta scope of `okta.myAccount.phone.manage` is required to use this endpoint.
+
+> **Note:** Admin users are not allowed to call the `/idp/myaccount/phones/{id}/verify` endpoint.
+
+#### Required scope and role
+
+#### Request path parameters
+
+| Parameter  | Type   | Description                                       |
+| ---------- | ------ | ------------------------------------------------- |
+| `id` | String | The id of the phone factor. Found in the response when a new phone number is created successfully (`POST /idp/myaccount/phones`) or phone(s) is retrieved (`GET /idp/myaccount/phones`)  |
+
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+This request requires the `verificationCode` property as its request body.
+
+| Property | Type                     | Description                          |
+| -------- | -------------------------|--------------------------------------|
+| `verificationCode`  | String | A 6-digit verification code that the user receives via `SMS` or `CALL` |
+
+#### Response body
+
+Returns an empty response with an HTTP 204 status code.
+
+Disabling the factor type of the corresponding method on the org returns a 403 FORBIDDEN with error code E0000038.
+Disabling the IDP_MY_ACCOUNT_API feature flag returns a 401 UNAUTHORIZED with error code E0000015.
+Passing an invalid `id` returns a 404 NOT FOUND with error code E0000008.
+Passing an invalid `verificationCode` returns a 400 BAD REQUEST with error code E0000001.
```

Comment:
This happens when we could not find a matching cached challenge from redis based on `factorId` and `verificationCode`.